python/gink/impl/coding.py

```diff
@@ -323,8 +324,8 @@ def entries_equiv(pair1: PlacementBuilderPair, pair2: PlacementBuilderPair) -> b
     assert pair1.placement.container == pair2.placement.container
     if pair1.builder.HasField("pointee"):
         if pair2.builder.HasField("pointee"):
-            pointee1 = Muid.create(builder=pair1.builder.pointee, context=pair1.placement)
-            pointee2 = Muid.create(builder=pair2.builder.pointee, context=pair2.placement)
+            pointee1 = Muid.create(builder=pair1.builder.pointee)
+            pointee2 = Muid.create(builder=pair2.builder.pointee)
```

Comment:
Placement context was not being used in Muid.create. Let me know if instead of removing the argument, we should have Muid.create utilize placements.

Reply:
Yeah this should probably be `pairX.placement.placer`.  The use case is if someone doesn't include the timestamp or medallion in the entry proto and it needs to be implied by where the entry proto has been found.